### ISSUE TITLE
Documentation - OLDER versions of Karpenter

### ISSUE THREAD

**Issue by hvithval** [aws/karpenter-provider-aws#5476]
### Description

**How can the docs be improved?**
I need the documentation for version 0.27.3 and I can not find it, it would be interesting to have documentation of older versions available somehow.

**Comment by jonathan-innis** [comment]
> it would be interesting to have documentation of older versions available somehow

Agreed. We were maintaining a ton of versions on the public, but it was causing problems for us with latency. You can still access the content directly by going to the tagged version of the docs in the repo: https://github.com/aws/karpenter-provider-aws/tree/v0.27.3/website/content/en/preview
